[ADD] Buscar produto no backend por ean13

lfdivino opened the pull request:
- Buscar o produto no backend ao clicar no botão 'Buscar +', em
caso positivo o produto retornado é adicionado ao objeto banco
de dados do ponto de venda, ficando assim disponível para
futuras pesquisas.

leorochael commented on l10n_br_pos/views/pos_template.xml:
```diff
@@ -20,13 +20,11 @@
-        <link rel="stylesheet" href="/l10n_br_pos/static/src/css/pos.css"/>
-    </xpath>
-</template>
+    <template id="pos_l10n_br_assets" name="pos_l10n_br_css" inherit_id="point_of_sale.index">
```
Esta mudança parece desnecessária nesse commit, só mudou whitespace.

Se ela for de fato desnecessária, será possível instalar essa mudança sem atualizar módulos. @lfdivino, confirma isso?

lfdivino replied:
Confirmo sim @leorochael, não precisa atualizar o módulo.

leorochael commented on l10n_br_pos/static/src/js/widgets.js:
```diff
@@ -109,4 +109,31 @@ function l10n_br_pos_widgets(instance, module){
+                    self.pos.db.add_products(res);
+                    self.perform_search(self.category, ean13_produto, event.which === 13);
+                } else {
+                    alert("Este produto não possui estoque no momento!");
```
Por que vc usa `alert()` aqui, mas usa `show_popup()` abaixo?

Se eu entendi direito a query, você está apenas buscando se o produto existe, não se ele tem estoque, mas a mensagem desse alert é "Este produto não possui estoque no momento").

Existe de fato uma checagem de estoque aqui que eu não percebi?

leorochael replied:
Outra coisa, para que serve o `event.which === 13` aqui?

lfdivino replied:
Na verdade na primeira versão eu tinha sim colocado uma verificação de estoque, mas o André Domingues me sugeriu deixar sem ela porquê depois seria mais fácil trabalhar com o inventário. Vou alterar a nomenclatura do retorno e vou coloca-lo em uma show_popup() também.

leorochael committed:
[ADD] Buscar produto no backend por ean13

- Buscar o produto no backend ao clicar no botão 'Buscar +', em
caso positivo o produto retornado é adicionado ao objeto banco
de dados do ponto de venda, ficando assim disponível para
futuras pesquisas.
